src/pages/monitoring/kafka/index.tsx

```diff
@@ -0,0 +1,21 @@
+import BarChart from "../../../components/visualisation/bar";
```

Comment:
the import chain is getting long ,so can you add one index.tsx in components folder and export all the component from there ,as I did for pages